src/fides/api/ctl/database/seed.py

```diff
@@ -385,3 +404,128 @@ async def load_default_resources(async_session: AsyncSession) -> None:
     await load_default_organization(async_session)
     await load_default_taxonomy(async_session)
     load_default_dsr_policies()
+
+
+async def load_samples(async_session: AsyncSession) -> None:
+    log.info("Loading sample resources into database...")
+    try:
+        sample_resources = load_sample_resources_from_project(strict=False)
+        for resource_type, resources in sample_resources.items():
+            await upsert_resources(
+                sql_model_map[resource_type],
+                [e.dict() for e in resources],
+                async_session,
+            )
+    except QueryError:  # pragma: no cover
+        pass  # The upsert_resources function will log any error
+
+    log.info("Loading sample connections into database...")
+    try:
+        sample_connections = load_sample_connections_from_project()
+        with sync_session() as db_session:
+            for connection in sample_connections:
+                # If the connection config already exists, skip creation!
+                # NOTE: This creates an edge case where the sample data was
+                # created previously, but has since changed. By not deleting &
+                # recreating here, we allow the "old" data to persist. That's an
+                # acceptable risk here, so we log an INFO message to provide a
+                # breadcrumb back to this code.
+                connection_config = ConnectionConfig.get_by(
+                    db=db_session, field="key", value=connection.key
+                )
+                if connection_config:
+                    log.info(
+                        f"Sample connection '{connection.key}' already exists, skipping..."
+                    )
+                    continue
+
+                # For SaaS connections, reuse our "instantiate from template" API
+                if (
+                    connection.connection_type == "saas"
+                    and connection.saas_connector_type
+                ):
+                    log.info(
+                        f"Loading sample connection with SaaS connection type '{connection.saas_connector_type}'..."
+                    )
+                    saas_template_data = dict(connection)
+                    saas_template_data["instance_key"] = connection.key
+                    saas_template_data.pop(
+                        "dataset", None
+                    )  # not supported by this API!
+                    instantiate_connection_from_template(
+                        db=db_session,
+                        saas_connector_type=connection.saas_connector_type,
+                        template_values=SaasConnectionTemplateValues.parse_obj(
+                            saas_template_data
+                        ),
+                    )
+
+                    # Check that it succeeded!
+                    connection_config = ConnectionConfig.get_by(
+                        db=db_session, field="key", value=connection.key
+                    )
```

Comment:
Wanted to make sure this is intentional - ignoring failures instead of trying to exit when something doesn't happen as expected.

I experimented with adding a consent connector, something I've wanted for awhile, and I had the wrong `secrets` type in `sample_connections.yml`, so even though all the secrets were populated, it was still missing a required secret and failed secrets validation here.  The logs were all very clear and it was easy to track down what I was missing, but I guess I was surprised everything came up properly, just missing the bad connector.

https://github.com/ethyca/fides/pull/3104/files

Reply:
Yes, I thought about this intentionally - the logs are there to provide the breadcrumbs for us, but I'd like that command to be pretty resilient against small misconfigurations. After all it'll be the _main_ command to use for all automated testing, so I'd prefer the whole environment still comes up but the failure is just that the connector doesn't appear. That's the difference between 1/100 failing tests and 100/100 failing tests when a small change like that is made 👍

Reply:
> After all it'll be the main command to use for all automated testing
makes sense